fv/jaspergold.tcl

```diff
@@ -31,17 +31,20 @@ proc cvfv_rerun {} {
   set_message -info VERI-2418
   ## Allow empty port in module declaration
   set_message -info VERI-8026
+  ## Allow multiplier blackboxing
+  set_message -info WNL018
 
   # Analyze & Elaborate
   analyze  -sv12  -f fv.flist
-  elaborate  -top uvmt_cv32e40x_tb
+  elaborate  -top uvmt_cv32e40x_tb  -extract_covergroup
 
   # Clock & Reset
   clock  clknrst_if.clk
   reset  ~clknrst_if.reset_n
 
   # Assumes
   assume  -from_assert  {*_memory_assert_i.u_assert.a_r_after_a}
+  assume  -from_assert  {*.obi_*_memory_assert_i.*.a_*par}
```

Comment:
Not 100% sure of this, could you take a look @silabs-robin ?

Reply:
Did it just appear when merging?
If yes, strange that it didn't appear before.

I don't think it is too bad.
It ensures parity signals on the OBI interface are correct, which doesn't matter for the X.

Reply:
Yeah, it did, hem, weird. Okei, sounds good then.